packages/flutter_tools/lib/src/observatory.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
We should start merging this functionality with `package:flutter_driver`. The overlap is too big. It's good that you are consolidating this. It will be much easier to refactor it later.

Logged https://github.com/flutter/flutter/issues/3893 with a proposal.